From c4929a544bc33c54e44fd9ea9ff9641c148310ea Mon Sep 17 00:00:00 2001
From: Terry Lv <r65388@freescale.com>
Date: Tue, 21 Jun 2011 14:29:52 +0800
Subject: [PATCH 0241/2463] ENGR00139235-1 include: fsl_device add clock name
 in platform data

Add clock name, max_volt, min_volt field

Signed-off-by: Terry Lv <r65388@freescale.com>
---
 include/linux/fsl_devices.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index 25a1970..ba5940d 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -268,6 +268,9 @@ struct mxc_iim_platform_data {
 struct mxc_otp_platform_data {
 	char    **fuse_name;
 	char    *regulator_name;
+	char    *clock_name;
+	unsigned int min_volt;
+	unsigned int max_volt;
 	unsigned int    fuse_num;
 };
 
-- 
1.7.7.4

